torch/quantization/fx/quantize.py

```diff
@@ -346,50 +344,47 @@ def _generate_qconfig_map(
             self,
             root: torch.nn.Module,
             input_graph: Graph,
-            qconfig_dict: Any) -> None:
-        global_qconfig = qconfig_dict.get('', None)
+            qconfig_dict: Any,
+            node_name_to_scope: Dict[str, Tuple[str, Any]]) -> None:
```

Comment:
any chance we can clarify `Any` further to the real type?  Motivation: if someone other than the code author is debugging this, as written there is no way to deduce the type other than looking up this PR and tracing what's happening.  If the type was specified, the code reader has a lower burden.

if it's challening to define the type for some reason, can we add a comment with an example value

Reply:
sure